packages/unplugin/src/sentry/logger.ts

```diff
@@ -0,0 +1,47 @@
+import { makeSentryClient } from "../sentry/telemetry";
+import { Options } from "../types";
+import { SeverityLevel } from "@sentry/node";
+
+export default function logger(props: Pick<Options, "silent" | "telemetry" | "org">) {
```

Comment:
Any chance we can pass the hub or client from `index.ts` here so that we don't have to create a new client in this function?

Reply:
If we do this, we can also get rid of the `"telemetry"` property in `props`.

Reply:
that sounds nice as I too don't really like that we have to re-instantiate client